pkg/scheduler/factory/factory.go

```diff
@@ -190,6 +188,12 @@ type configFactory struct {
 	// framework has a set of plugins and the context used for running them.
 	framework framework.Framework
 
+	// TODO(#80216): Remove pod and node listers.
```

Comment:
same here

Reply:
To keep them visible in preparation for the TODO.

Reply:
Why not add a TODO before line 168?

Reply:
I agree with @draveness. Why are we moving these lines?

Reply:
Done